suite-native/module-accounts-import/src/accountsImportSlice.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export const actionPrefix = '@accountsImport';
+
+export type AccountsImportState = Record<string, string>;
```

Comment:
What is `string, string`? Can you please create type aliases with proper names? I thing we already have `AccountHash` type somewhere defined